src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Security/AuthorizationHandlers/UpdateSpecificMethodologyAuthorizationHandlersTests.cs

```diff
@@ -0,0 +1,175 @@
+﻿using System;
+using System.Collections.Generic;
+using GovUk.Education.ExploreEducationStatistics.Admin.Security.AuthorizationHandlers;
+using GovUk.Education.ExploreEducationStatistics.Admin.Services.Methodologies;
+using GovUk.Education.ExploreEducationStatistics.Admin.Tests.Services;
+using GovUk.Education.ExploreEducationStatistics.Content.Model;
+using Microsoft.AspNetCore.Authorization;
+using Xunit;
+using static GovUk.Education.ExploreEducationStatistics.Admin.Security.SecurityClaimTypes;
+using static GovUk.Education.ExploreEducationStatistics.Admin.Tests.Security.AuthorizationHandlers.Utils.
+    ReleaseAuthorizationHandlersTestUtil;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Tests.Security.AuthorizationHandlers
+{
+    public class UpdateSpecificMethodologyAuthorizationHandlersTests
+    {
+        [Fact]
+        public void UpdateAllSpecificMethodologiesAuthorizationHandler()
+        {
+            AssertReleaseHandlerSucceedsWithCorrectClaims<UpdateSpecificMethodologyRequirement>(
+                new UpdateAllSpecificMethodologiesAuthorizationHandler(),
+                UpdateAllMethodologies
+            );
+        }
+
+        [Fact]
+        public async void HasNonPrereleaseRoleOnAnyAssociatedReleaseAuthorizationHandler_Succeed()
+        {
+            var userId = Guid.NewGuid();
+            var publication = new Publication
+            {
+                Id = Guid.NewGuid()
+            };
+            var release = new Release
+            {
+                Id = Guid.NewGuid(),
+                PublicationId = publication.Id
+            };
+            var userReleaseRole = new UserReleaseRole
+            {
+                UserId = userId,
+                Release = release,
+                Role = ReleaseRole.Lead
+            };
+            var methodology = new Methodology
+            {
+                Id = Guid.NewGuid(),
+                Publications = new List<Publication> { publication }
+            };
+
+            var contentDbContextId = Guid.NewGuid().ToString();
+            await using (var contentDbContext = DbUtils.InMemoryApplicationDbContext(contentDbContextId))
+            {
+                await contentDbContext.AddAsync(publication);
+                await contentDbContext.AddAsync(release);
```

Comment:
We don't need to save every single entity explicitly. Entities that are related to one another get persisted in a recursive way. In this case, `userReleaseRole` would already persist the release, so you wouldn't need to do it here too.